scripts/app.ts

```diff
@@ -257,7 +267,5 @@ var actionProvider = {
     }
 };
 
-
-// Register context menu action provider
-VSS.register("blueprint.vsts-extension-split-work.vsts-extension-split-work-action", actionProvider);
-VSS.register("vsts-extension-split-work-action", actionProvider);
\ No newline at end of file
+VSS.register(VSS.getContribution().id, actionProvider);
+VSS.notifyLoadSucceeded();
```

Comment:
I am not familiar with the original author's intent in calling vss.register twice, but I assume with this change there is no side effects or removing one.

Reply:
@davidcorrigan714 This used to register the "vsts-extension-split-work-**action**". Now both this file and the dialog.tsx register "vsts-extension-split-work-**dialog**".  Shouldn't this one still be action? How does the menu action load? I still see it defined as  contribution in the vss-extension.json